tests/spec/core/link-to-dfn-spec.js

```diff
@@ -44,4 +44,21 @@ describe("Core — Link to definitions", function() {
     expect(noCodeWrap.querySelector("code")).toBeFalsy();
     expect(noCodeWrap.textContent).toEqual("the request interface");
   });
+
+  it("checks for duplicate definitions", async () => {
```

Comment:
Ok, this initial test looks great.

Couple of other things we also want to check (looking at the code). 

 1. Does it automatically generate an id? 
 1. If there is an `id` attribute already present, make sure it doesn't trash it. 
 1. If there is an `title` attribute already present, make sure it doesn't trash it. The current code actually has bug, because it does trash it.

Reply:
If you can think of other things, that would be great.

Reply:
On it!